data/upgrades/astromech.json

```diff
@@ -651,5 +651,129 @@
     "standard": true,
     "extended": true,
     "epic": true
+  },
+  {
+    "name": "R2-A3",
+    "limited": 1,
+    "xws": "r2a3-battleofyavin",
```

Comment:
I generally put the -battleofyavin suffix on these upgrades and pilots, even when no "regular" version exists already. I think this breaks xws norms, but YASB is already doing this, and I think it makes sense for this style of card to have the suffix on all cards for clarity. Any strong opinions?

Reply:
Put it on. Aligning with yasb streamlines things, and I was leaning that way already.